WordPress/Classes/ViewRelated/Me/App Settings/About/UnifiedAboutViewController.swift

```diff
@@ -59,11 +60,32 @@ class UnifiedAboutViewController: UIViewController {
         ]
     ]
 
+    let headerView: UIView = {
+        // These customizations are temporarily here, but if this VC is moved into a framework we'll need to move them
+        // into the main App.
+        let appInfo = UnifiedAboutHeaderView.AppInfo(
+            icon: UIImage(named: AppIcon.currentOrDefault.imageName) ?? UIImage(),
+            name: (Bundle.main.object(forInfoDictionaryKey: "CFBundleDisplayName") as? String) ?? "",
+            version: Bundle.main.detailedVersionNumber() ?? "")
+
+        let fonts = UnifiedAboutHeaderView.Fonts(
+            appName: WPStyleGuide.serifFontForTextStyle(.largeTitle, fontWeight: .semibold),
+            appVersion: WPStyleGuide.tableviewTextFont())
+
+        return UnifiedAboutHeaderView(appInfo: appInfo, fonts: fonts)
+    }()
+
     override func viewDidLoad() {
         super.viewDidLoad()
 
         let tableView = UITableView(frame: .zero, style: .insetGrouped)
         tableView.translatesAutoresizingMaskIntoConstraints = false
+
+        headerView.frame.size = headerView.systemLayoutSizeFitting(
```

Comment:
Could we maybe extract these lines into a small `updateHeaderView` or `resizeHeaderView` or similar method?

Reply:
Yeah will do.

Reply:
Moved this to the view.